requirements.txt

```diff
@@ -15,7 +15,7 @@ pyradiomics
 pyside6==6.2.0
 vtk
 matplotlib
-shapely>=1.8.1
+shapely>=1.8.1,<=1.8.5
```

Comment:
<2.0

Might be more bug fixes to 1.8
semver approach suggest that a major version change is the interface break point

Reply:
build pass and test pass when `shapely>=1.8.1,<2.0`